server/src/test/java/io/crate/expression/scalar/AgeFunctionTest.java

```diff
@@ -89,6 +91,8 @@ public void test_age_is_1_month_regardless_of_days_in_month() {
     }
 
     @Test
+    @Repeat(iterations = 100)
```

Comment:
With 10 reputations failed once, with 100 repetitions 11 failures. 
No failures in 100 repetitions with DateTimeZone.UTC in Interval. (will try out with 1000 repetitions)